recipes-bsp/u-boot/files/ae350-ax45mp/0001-Fix-SD-detect-failure.patch

```diff
@@ -0,0 +1,31 @@
+From 51d891dafceef1db0c0c01f6b1a45d7ce41846c8 Mon Sep 17 00:00:00 2001
+From: Yu Chien Peter Lin <peterlin@andestech.com>
+Date: Fri, 5 Aug 2022 15:52:23 +0800
+Subject: [PATCH] Fix SD detect failure
+
+This patch fixes SD detection for bitmaps without high speed mode
+support.
+
+Signed-off-by: Yu Chien Peter Lin <peterlin@andestech.com>
```

Comment:
Can you add an upstream status?

Reply:
It won't be upstream, but I've removed mmc property `cap-sd-highspeed` from kernel dts, so no need to keep this patch.